imapautofiler/secrets.py

```diff
@@ -67,6 +76,17 @@ def configure_providers(cfg):
             username=cfg['server']['username'],
         )
 
+    # Third, we will try for a env variable password if configured
+    try:
+        use_env_variable = cfg['server']['use_env_variable']
+    except KeyError:
+        use_env_variable = False
+
+    if use_env_variable:
+        LOG.debug("Password configured from ENV variable")
```

Comment:
This message could show the variable name, so that if there is a problem the user knows exactly where the app is looking.